src/client/java/teammates/client/scripts/sql/SeedDb.java

```diff
@@ -223,6 +236,103 @@ feedbackSessionCreatorEmail, feedbackSessionInstructions, getRandomInstant(),
         }
     }
 
+    private void seedFeedbackQuestions(int courseNumber, String courseId) {
+        assert MAX_FEEDBACKSESSION_FOR_EACH_COURSE_SIZE <= MAX_QUESTION_PER_COURSE;
```

Comment:
I chose to define the limits based on where the method is being called. In this case, since `seedFeedbackQuestions` is called by `seedCourseEntities` instead of `seedFeedbackSession`, I based the limit on course instead. 

We could change it to cascade from `seedFeedbackSession` instead if we think its clearer too?

Reply:
We can leave it as such then